### DISCUSSION TITLE
Raise css selector priority

### DISCUSSION THREAD

**Discussion by VasilNikolov** [ant-design/ant-design#50888] [General]
### What problem does this feature solve?

Is there a recommended approach to raise the css selector specificity of all ant styles? In the docs there's a mention of using postcss plugins like [postcss-add-root-selector](https://www.npmjs.com/package/postcss-add-root-selector) , but I think it's in a different context, since it doesn't prefix any of the ant styles, only the ones imported by the app.

### What does the proposed API look like?

For example currently prefixCls changes the default ant classes and the styles for them, but an option for adding a prefix selector to the styles only, without affecting the generated classes should be useful.

**Comment by afc163** [comment]
Try https://ant.design/docs/react/compatible-style#layer

**Reply by VasilNikolov** [reply]
This is supposed to reduce the specificity? I'm looking for an increase, for example the default output is 

.ant-flex { //some styles }

I want to change it to:

.my-app .ant-flex { //some styles }

**Reply by VasilNikolov** [reply]
@afc163 Can find a way to increase the priority. You can only separate other styling layers to prevent conflicts with third party libraries. But unfortunately we don't have control over the other styles, only the ant ones.
